packages/tupaia-web/src/api/queries/useMapOverlays.ts

```diff
@@ -48,7 +48,7 @@ const mapOverlayByCode = (
  */
 export const useMapOverlays = (projectCode?: ProjectCode, entityCode?: EntityCode) => {
   const [urlSearchParams] = useSearchParams();
-  const { data, isLoading, error } = useQuery(
+  const { data, isLoading, error, isFetched } = useQuery(
```

Comment:
I was looking like something like this a while ago and couldn't find it so good find